firefox_ui_tests/functional/locationbar/test_access_locationbar.py

```diff
@@ -27,6 +26,7 @@ def setUp(self):
         self.autocomplete_results = self.locationbar.autocomplete_results
         self.urlbar = self.locationbar.urlbar
 
+    @skip_under_xvfb
     def test_access_locationbar_history(self):
```

Comment:
Probably this could be related to [bug 1038614](https://bugzilla.mozilla.org/show_bug.cgi?id=1038614).